core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1657,6 +1657,16 @@
 
   public static final String CARBON_SEARCH_MODE_ENABLE_DEFAULT = "false";
 
+  /**
+   * If set to true, SDK enabled.
+   * Default value is false, SDK not enable.
+   */
+  @CarbonProperty
+  @InterfaceStability.Unstable
+  public static final String CARBON_SDK_ENABLE = "carbon.sdk.enabled";
```

Comment:
What does SDK enabled mean?

Reply:
It's constant, which can  show the status of SDK.